backend/test/institution_hierarchy_handler_test.py

```diff
@@ -104,7 +103,8 @@ def initModels(cls):
     cls.otherinst.parent_institution = cls.institution.key
     cls.otherinst.put()
     cls.otheruser.institutions_admin = [cls.otherinst.key]
-    cls.otheruser.add_permission("remove_link", cls.otherinst.key.urlsafe())
+    cls.otheruser.add_permission("remove_link", cls.institution.key.urlsafe())
```

Comment:
Remove initModels() function and use mocks module to create object mocks for each test.